src/components/OpenSource.jsx

```diff
@@ -72,29 +64,15 @@ const Contribution = (props) => {
 
 const OpenSource = () => {
   const [contributions, setContributions] = useState([]);
-  const [filterContribution, setFilterContribution] = useState([]);
-  const [activeFilter, setActiveFilter] = useState("All");
 
   useEffect(() => {
-    setContributions(openSourceContributions);
-    setFilterContribution(openSourceContributions);
```

Comment:
Wait, why have we removed this?

We still want to keep this filter but the filter is based on the name of the repo.

Reply:
I've added them back